test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Is this committed by mistake?

Reply:
It will be removed;

```sh
0ee78c67 Removed accidentally committed test.sh :D
 test.sh | 12 ------------
 1 file changed, 12 deletions(-)
d0ebe643 Fixed erroneous \
 test.sh | 12 ++++++++++++
 1 file changed, 12 insertions(+)
```

Reply:
Are you sure? Because https://github.com/netdata/netdata/pull/7949/commits/8a4f540610d72a1b4510fffe66ebb27cb6684623 adds it again

Reply:
It should conflict when I rebase :) But to be sure I'll add another commit :)

Reply:
Done :)